### PR TITLE
Add Storybook

### LINKED ISSUE #4349: Add Storybook
https://storybook.js.org/ will help us 

* document our design system and important ui components
* be a nice development environment for new components
* help us think through how to use replay to look into visual regressions in CI

### PR DESCRIPTION
**Pull request by jcmorrow**
### Changes

- Add Storybook and dependencies. This was not that bad! I wrestled with PostCSS for a while, but ultimately I just had to get everything on the right version number. I wish there had been more of a "gotchya" moment in this struggle, but it was really just the nth time I ran `npm install` and it worked.
- move some css variables up to `:root`. This makes sure that those variables are also present in Storybook.
- Add stories for network monitor. Notice that I export by default the connected request table, but I also export the non-connected RequestTable component, because that is easier to plug into Storybook.
- In the future, we'll probably want to *slightly* consolidate the number of elements that have to be connected, because that makes prototyping very fast, but the long-term plan should be to properly setup a Storybook Redux store (probably just a partial one) and get our Apollo mocks into a place where they can be used in Storybook as well as our tests.

Fixes #4349

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -2,20 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-var mapUrl = require("postcss-url-mapper");
```
Comment (jasonLaster) [inline postcss.config.js:5]: love seeing this cruft deleted

**Review by jasonLaster — APPROVED** [review]
Looking good! It's much easier to see this stuff in its own PR. Especially the postcss stuff

File: postcss.config.js (hunk 1 above)
Comment (jcmorrow) [reply postcss.config.js:5]: You know you can always count on me to find the unused stuff 🔨

Comment (jasonLaster) [reply postcss.config.js:5]: there are stories in that code! 😉

**Review by colbyr — COMMENTED** [review]
Just tried this out! looks awesome 👏

**Pull request merged**